students/david_russo/lesson3/mailroom.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+from sys import exit
+
+donors = [["Walter White", "Ray Donovan", "John Snow", "Walter White", "Ray Donovan", "John Snow", "Walter White", "Ray Donovan", "John Snow", "Walter White", "Ray Donovan", "John Snow", "Walter White", "Ray Donovan", "John Snow"],
+          [10000, 2500, 10, 15000, 6500, 5, 20000, 10500, 3, 25000, 15000, 2, 30000, 19500, 1]]
+
+# define a function to prompt the user for their desired action
+def prompt_user():
+    print("What would you like to do? Your options are: 1) Send a Thank You, 2) Create a Report, 3) quit")
+    response = input("Please type your selection > ")
+    while(response not in ("Send a Thank You", "Create a Report", "quit")):
+        print("Please enter one of 'Send a Thank You', 'Create a Report', or 'quit' > ")
+        response = input("Please type your selection > ")
+    return response
```

Comment:
So my first assumption was that a user should enter the number of what they wanted to do, not spell it out completely.  You might want to remove the numbers in your first print statement.

Reply:
I would also use the following:
```
while(response.lower() not in ("send a thank you", "create a report", "quit")):
```
so you don't have to worry about casing.